plugins/module_utils/network/sonic/utils/formatted_diff_utils.py

```diff
@@ -509,31 +566,27 @@ def derive_config_from_deleted_cmd_dict(command, exist_conf, test_keys=None, key
     return key_matched, new_conf
 
 
-def get_formatted_config_diff(exist_conf, new_conf):
+def get_formatted_config_diff(exist_conf, new_conf, verbosity=0):
 
-    diff_correction = [
-        {'python_str': ': None', 'ansible_str': ': null'},
-        {'python_str': ': True', 'ansible_str': ': true'},
-        {'python_str': ': False', 'ansible_str': ': false'},
-    ]
+    exist_conf = json.dumps(exist_conf, sort_keys=True, indent=4, separators=(u',', u': ')) + u'\n'
+    new_conf = json.dumps(new_conf, sort_keys=True, indent=4, separators=(u',', u': ')) + u'\n'
 
-    bfr = pformat(exist_conf)
-    for d_correct in diff_correction:
-        bfr = bfr.replace(d_correct['python_str'], d_correct['ansible_str'])
+    bfr = exist_conf.replace("\"", "\'")
+    aft = new_conf.replace("\"", "\'")
 
-    aft = pformat(new_conf)
-    for d_correct in diff_correction:
-        aft = aft.replace(d_correct['python_str'], d_correct['ansible_str'])
+    bfr_list = bfr.splitlines(True)
+    aft_list = aft.splitlines(True)
+    diffs = context_diff(bfr_list, aft_list, fromfile='before', tofile='after')
 
-    bfr_list = list(bfr.split(',\n'))
-    aft_list = list(aft.split(',\n'))
-    diffs = context_diff(bfr_list, aft_list,
-                         fromfile='before_config',
-                         tofile='after_config')
     formatted_diff = list()
     for diff in diffs:
-        if diff.endswith('\n'):
-            diff = diff.rstrip('\n')
+        if verbosity >= 3:
+            if diff.endswith('\n'):
+                diff = diff.rstrip('\n')
         formatted_diff.append(diff)
 
+    if verbosity < 3:
+        fmt_diff = u''.join(formatted_diff)
+        formatted_diff = {'prepared': fmt_diff}
```

Comment:
Could be simplified based on verbosity:
```python
if verbosity >= 3:
    formatted_diff = []
    for diff in diffs:
        formatted_diff.append(diff.rstrip('\n'))
else:
    formatted_diff = {'prepared': u''.join(diffs)}
```

Reply:
Done this. Thank you very much!

Reply:
This change isn't visible yet in the version of this file on the PR branch. Please commit/push the change to resolve this issue.